### PR TITLE
add few packages that help in operations

### PR DESCRIPTION
**Pull request by fgeorgatos**
probably you might want to keep only `less` from this list - preferences?

### CODE REVIEW

**Comment by jirikuncar** [comment]
@fgeorgatos can you create your custom operation image based on this base image and simply run it only when necessary?

**Comment by rokroskar** [comment]
I think `less` is a reasonable addition, not sure about the others

File: docker/base/Dockerfile [hunk 1]
```diff
@@ -25,6 +25,10 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
     unzip \
     vim \
     musl-dev \
+    less \
+    lynx \
+    links \
+    lftp \
```
[the remark this answers is not in the document]
Comment (fgeorgatos) [reply docker/base/Dockerfile]: i added them as common downloaders of data, not so much as text browsers.
I've hit cases in the past where any of these tools are needed for the occasional data transfer and one does not automatically replace the other. Feel free to choose what you like!

[the remark this answers is not in the document]
Comment (fgeorgatos) [reply docker/base/Dockerfile]: OK, I trimmed down to `less`, there is only one choice left: merge/reject.

**Pull request merged**